app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -111,10 +126,22 @@ export default class EditStreamInfo extends Vue {
     }
   }
 
+  getAvailableProfiles () {
```

Comment:
I would call this `loadAvailableProfiles`, since `get` kind of implies it should return something instead of just setting a value on the component.

Reply:
Done.